p5js/Pix2Pix_callback/sketch.js

```diff
@@ -61,13 +67,17 @@ function transfer() {
 
   // Select canvas DOM element
   let canvasElement = document.getElementById('defaultCanvas0');
```

Comment:
I might suggest using p5.dom for the p5 examples? Would `inputCanvas.elt` get you the element you are looking for or no? In general I would avoid using `'defaultCanvas0'` in code and explicitly assign an `id` to be more clear.